### ISSUE TITLE
How do I use QSV for auto conversions

### ISSUE THREAD

**Issue by rubyredrick** [jlesage/docker-handbrake#96]
I have set up qsv on my Synology DS918+ and it works from the handbrake web interface.

I can’t seem to figure out how to get this to work for auto conversion. I guess this needs a change to my docker run command, but I can’t figure out what the changes are.

**Comment by etreml** [comment]
Hi rubyredrick,
great I can help a fellow DS918+ friend ;-)

I've made my own conversion preset (wirt h265 qsv, quality settings, audio settings, ...) and use this conversion preset in the docker enviroment settings

![image](https://user-images.githubusercontent.com/4254786/72140991-3d9a8500-3392-11ea-8874-cd0dd00587ac.png) (sorry, german settings)

**Comment by rubyredrick** [comment]
> Hi rubyredrick,
> great I can help a fellow DS918+ friend ;-)
> 
> I've made my own conversion preset (wirt h265 qsv, quality settings, audio settings, ...) and use this conversion preset in the docker enviroment settings
> 
> ![image](https://user-images.githubusercontent.com/4254786/72140991-3d9a8500-3392-11ea-8874-cd0dd00587ac.png)

Thanks. but your link only leads to a jpg image.

The problem I'm having is that I'm ending up with videos which have a black screen, and only audio.  This is my current setup.

I also made a preset. 
[QSV 1080p30.json.txt](https://github.com/jlesage/docker-handbrake/files/4047868/QSV.1080p30.json.txt) Note I renamed it from .json to .json.txt so that GitHub would accept it as an attachment.

This is my run command:
`sudo docker run -d \
    --name=videoproc \
    --privileged \
    -e HANDBRAKE_DEBUG=1 \
    -e USER_ID=0 \
    -e GROUP_ID=0 \
    -e AUTOMATED_CONVERSION_HANDBRAKE_CUSTOM_ARGS="--enable-qsv-decoding" \
    -e AUTOMATED_CONVERSION_PRESET="QSV 1080p30" \
    -e HANDBRAKE_DEBUG=1 \
    -p 5800:5800 \
    -v /volume1/docker/appdata/videoproc:/config:rw \
    -v $HOME:/storage:ro \
    -v /volume1/video/downloads/completed:/watch:rw \
    -v /volume1/video/downloads/converted:/output:rw \
    -v /volume1/video/downloads/:/downloads:rw \
    -v /volume1/video/:/video:rw \
    --device=/dev/dri:/dev/dri \
    rick/videoprocessor
`
IIRC when I just had the preset it was failing so I adde the --enable-qsv-decoding flag.

Here is my handbrake debug log 
[handbrake.debug.log](https://github.com/jlesage/docker-handbrake/files/4047877/handbrake.debug.log)

**Comment by jlesage** [comment]
Are you using the same preset that is working in the UI?

**Comment by rubyredrick** [comment]
Yes, I just reran using the web UI and it’s fine. I’m mystified.
